node/sc/vt_recovery_test.go

```diff
@@ -597,10 +603,7 @@ func prepare(t *testing.T, vtcallback func(*testInfo)) *testInfo {
 			case ev := <-recoveryCh:
 				isRecovery = ev
 			case ev := <-requestVTCh:
-				// Intentionally lost a single handle value transfer.
-				// Since the increase of monotony in nonce is checked in the contract,
-				// all subsequent handle transfer will be failed.
-				if ev.RequestNonce == (testTxCount - testPendingCount) {
+				if ev.RequestNonce >= (testTxCount - testPendingCount) {
```

Comment:
(testTxCount - testPendingCount) can not be negative? If a negative number is available, the type will be uint64 underflow.

Reply:
Negative numbers cannot be obtained unless the test code is deliberately modified to be negative.